packages/react/src/transactions/adapters/__tests__/PublicationCallGateway.spec.ts

```diff
@@ -82,6 +90,42 @@ function mockCreatePostViaDispatcherMutationMockedResponse({
   };
 }
 
+function mockCreateCommentTypedDataMutationMockedResponse({
+  variables,
+  data,
+}: {
+  variables: CreateCommentTypedDataMutationVariables;
+  data: CreateCommentTypedDataMutation;
+}): MockedResponse<CreateCommentTypedDataMutation> {
+  return {
+    request: {
+      query: CreateCommentTypedDataDocument,
+      variables,
+    },
+    result: {
+      data,
+    },
+  };
+}
+
+function mockCreateCommentViaDispatcherMutationMockedResponse({
+  variables,
```

Comment:
these two mock functions, is it fine to keep them here or they should be places in `@lens-protocol/api-bindings/mocks` as all other mocks?

Reply:
moved